benchto-driver/src/test/resources/application.yaml

```diff
@@ -23,6 +23,7 @@ presto:
 graphite:
   url: http://graphite:18088
   resolution.seconds: 1
+  metrics-delay.seconds: 0
```

Comment:
This is must here to have tests single-threaded. The REST mocking don't work otherwise.